### DISCUSSION TITLE
When the driver file or system is updated, ghelper often resets itself

### DISCUSSION THREAD

**Discussion by EEErink** [seerge/g-helper#2008] [Ideas]
Can we export the configuration file so that we can import our own configuration file when Ghelper unexpectedly resets?
Profiles can also be communicated with users of the same model

**Comment by seerge** [comment]
@EEErink hello.

What kind of driver or system update is that ? App stores it's settings under ``%APPDATA%\GHelper\config.json``. And it already does backups of this config, so it should not reset anyhow.

If you reset windows or erase that settings by some reason, you should just make a manual back-up of that file :)

**Reply by EEErink** [reply]
such as Amd GPU driver or Win update,
THANK U , i'll try to back-up it
